LBL/Lbl/Lbl.Server/Controllers/BaseQueryController.cs

```diff
@@ -14,6 +14,7 @@ namespace Lbl.Server.Controllers
 
     public class BaseQueryController<T, TR, TV> : ApiController where T : Entity where TR : BaseRequestModel<T> where TV : BaseViewModel<T>
     {
+        [AllowAnonymous]
```

Comment:
its not  right. because all of the search method of all controllers now will be publicly accessible.